ethcore/wasm/src/runtime.rs

```
@@ -724,6 +725,26 @@ impl<'a> Runtime<'a> {
 
 		Ok(())
 	}
+
+	/// Signature: `fn request_bytes(key: *const u8, result: *mut u8)`
+	pub fn request_bytes(&mut self, args: RuntimeArgs) -> Result<()> {
+		let key = self.h256_at(args.nth_checked(0)?)?;
+		let bytes = self.ext.request_bytes(key).unwrap();
+		self.memory.set(args.nth_checked(1)?, &bytes);
+
+		Ok(())
+	}
+
+	/// Signature: `fn store_bytes(bytes: *const u8, len: u64, key: *mut u8)`
+	pub fn store_bytes(&mut self, args: RuntimeArgs) -> Result<()> {
+		let bytes_ptr: u32 = args.nth_checked(0)?;
+		let len: u64 = args.nth_checked(1)?;
+		let bytes = self.memory.get(bytes_ptr, len as usize)?;
+		let key = self.ext.store_bytes(&bytes).unwrap();
```

no unwrap. this should return an error

---

fixed